include/nbl/asset/utils/IMeshPacker.h

```diff
@@ -214,15 +214,14 @@ class IMeshPacker : public IMeshPackerBase
 
     struct IdxBufferParams
     {
-        SBufferBinding<ICPUBuffer> idxBuffer;
-        uint32_t idxCnt = 0u;
+        SBufferBinding<ICPUBuffer> idxBuffer = { 0u, nullptr };
         E_INDEX_TYPE indexType = EIT_UNKNOWN;
     };
 
     //TODO: functions: constructTriangleBatches, convertIdxBufferToTriangles, deinterleaveAndCopyAttribute and deinterleaveAndCopyPerInstanceAttribute
     //will not work with IGPUMeshBuffer as MeshBufferType, move it to new `ICPUMeshPacker`
 
-    TriangleBatches constructTriangleBatches(const MeshBufferType* meshBuffer, IdxBufferParams idxBufferParams, core::aabbox3df* aabbs) const
+    TriangleBatches constructTriangleBatches(MeshBufferType* meshBuffer, IdxBufferParams idxBufferParams, core::aabbox3df* aabbs) const
```

Comment:
why did you remove const?

Reply:
https://github.com/Devsh-Graphics-Programming/Nabla/pull/111/files/3e94ece32799e431842f15e4768741d7dbfa95b8..16f6abdb8bf00336e2b0da3c9cbcb4df8cbf64fe#diff-9da5291f7aea536f7b8a62660d1bf42cc8b70620472e0b385f6775a789e4d636R276-R282